.travis.yml

```diff
@@ -1,24 +1,29 @@
 language: python
 cache: pip
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
+matrix:
+  include:
+    - python: 2.7
+    - python: 3.5
+    - python: 3.6
+    - python: 3.7
+      dist: xenial    # required for Python >= 3.7 (travis-ci/travis-ci#9069)
```

Comment:
The __matrix:__ and __dist: xenial__ is no longer required because Xenial is now the default distro on Travis CI.
https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment